### DISCUSSION TITLE
How to use Edge browser?

### DISCUSSION THREAD

**Discussion by chenbimo** [wxt-dev/wxt#183] [Q&A] [answered]
I tried using browser:edge but it still opened chrome

**Comment by aklinker1** [comment, marked as answer]
https://wxt.dev/guide/development.html#configure-browser-startup

WXT uses [`web-ext`](https://github.com/mozilla/web-ext) to open the browser in dev mode. It only knows where Chrome and Firefox are located by default. When using a different browser, you have to tell `web-ext` where that browser's binary or executable is located, otherwise it will open chrome by default (since most browsers are chromium and will behave the exact same).

You can do this by using the [`runner` config](https://wxt.dev/api/wxt/interfaces/InlineConfig.html#runner) inside `wxt.config.ts` if you want to commit the config, or in a separate, gitignored `web-ext.config.ts` file if not.

```ts
// ~/web-ext.config.ts or <root>/web-ext.config.ts
import { defineRunnerConfig } from 'wxt';

export default defineRunnerConfig({
  binaries: {
    // Customize which binary to use for a specific browser
    edge: "/path/to/edge"
  },
});
```

I'll add docs to [this page](https://wxt.dev/guide/multiple-browsers.html#target-browser) to explain this.

**Reply by aklinker1** [reply]
I've updated the docs: https://github.com/wxt-dev/wxt/commit/d9e9b43f8d2784f18bb1e4253170d1a092699a38

**Reply by chenbimo** [reply]
Ok!Very thinks!
